plugins/janus_streaming.c

```diff
@@ -1363,6 +1364,9 @@ int janus_streaming_init(janus_callbacks *callback, const char *config_path) {
 		if(!notify_events && callback->events_is_enabled()) {
 			JANUS_LOG(LOG_WARN, "Notification of events to handlers disabled for %s\n", JANUS_STREAMING_NAME);
 		}
+		janus_config_item *rtsp_port_range_key = janus_config_get(config, config_general, janus_config_type_item, "rtsp_client_port_range");
+		if(rtsp_port_range_key != NULL && rtsp_port_range_key->value != NULL)
+			rtsp_client_port_range = g_strdup(rtsp_port_range_key->value);
```

Comment:
This looks very inefficient. You're basically parsing the same setting over and over again, any time an RTSP mountpoint is created. Please look at how the SIP/SIPre/NoSIP plugins do it instead.

Reply:
I was following the `admin_key` example, since it is also defined in `general` section of the module, but I agree that keeping a string is inefficient and it would be better to cache the actual min and max port numbers.